man/login.1.xml

```diff
@@ -277,7 +277,7 @@
       &CONSOLE;
       &CONSOLE_GROUPS;
       &DEFAULT_HOME;
-      <phrase condition="no_pam">&ENV_HZ;</phrase>
```

Comment:
what's the reason why you remove the condition here?

Reply:
The condition is now part of ENV_HZ.xml, i.e. automatically included. The same is true for the other PAM-only XML-files in login.conf.d.

Reply:
Oh, I see - thanks.